epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModelGroup.java

```diff
@@ -301,10 +307,14 @@ protected boolean useViewStubLayoutParams(EpoxyModel<?> model, int modelPosition
 
   @Override
   protected final Holder createNewHolder() {
-    return new Holder();
+    Holder holder = new Holder();
+    holder.setModelGroup(this);
```

Comment:
`bindView` is only called once, when the Holder is first created. that means we only need the model group then, and not on all of the bind calls, so all this can be simplified. maybe the model group can be passed to the holder in the constructor

Reply:
I think should pass the model group to Holder.bindView, then the Holder instance will not hold the model group instance. In current version, the Holder instance hold the model group which create it forever, no chance to release it.

Reply:
> `bindView` is only called once, when the Holder is first created. that means we only need the model group then, and not on all of the bind calls, so all this can be simplified.

Yes.